src/forms/CurrentIncome.js

```diff
@@ -66,53 +66,67 @@ const IncomeForm = function ({ current, time, setClientProperty }) {
       <IntervalColumnHeadings type={ type } />
 
       {/* All kinds of things need to be explained. */}
-      {/* @todo Change 'labelInfo' to visible blurb at top */}
+      {/* @todo Change 'labelInfo' to visible blurb at top - delete below labelInfo? */}
+      
       <CashFlowRow
         { ...sharedProps }
         generic='earned'
         labelInfo='(Weekly income = hourly wage times average number of work hours per week)'>
-          Earned income
+        { snippets.earnedIncome.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='TAFDC'> TAFDC
+        generic='TAFDC'> 
+        { snippets.TAFDC.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='SSI'> SSI
+        generic='SSI'> 
+        { snippets.SSI.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='SSDI'> SSDI
+        generic='SSDI'>
+        { snippets.SSI.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='childSupportIn'> Child support received
+        generic='childSupportIn'>
+        { snippets.childSupport.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='unemployment'> Unemployment
+        generic='unemployment'> 
+        { snippets.unemployment.label }      
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='workersComp'> Worker’s comp
+        generic='workersComp'> 
+        { snippets.workersComp.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='pension'> Pension
+        generic='pension'>
+        { snippets.pension.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='socialSecurity'> Social security
+        generic='socialSecurity'>
+        { snippets.socialSecurity.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='alimony'> Alimony
+        generic='alimony'> 
+        { snippets.alimony.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='otherIncome'> Other income
+        generic='otherIncome'>
+        { snippets.otherIncome.label }
       </CashFlowRow>
+      {/** @todo Change "This prototype..." to snippets.incomePrototypeCalcuationHelp after changing en.js 
+        * This message is buried.  What does it mean?
+      */ }
```

Comment:
I believe it's supposed to reassure users who are expecting to input their SNAP benefit amount here, but can't because we're calculating it based on other things. We should probably make it look more like a "note" rather than a random line of text, though.

Can you add this in `en.js`?

Reply:
When you feel you've resolved this, @ethanstrominger , go ahead and remove this question.